examples/with-typescript/pages/detail.tsx

```diff
@@ -1,23 +1,20 @@
 import * as React from 'react'
-import { NextContext } from 'next'
+import { IContext } from 'next'
```

Comment:
Can this be `NextPageContext` instead ?

Reply:
`IContext` is internally used and only exported for the case where a user wants to extend the interface for all the codebase, but `NextPageContext` should be the way to go for 95% of the cases

Reply:
Another option could be to rename `IContext` to `NextPageContext`, yeah that would probably be better

Reply:
Yea, It's not imported properly so I used only IContex, but its good point that we should use `NextPageContext`